spanner/integration_test.go

```diff
@@ -2512,6 +2513,324 @@ func TestIntegration_BasicTypes(t *testing.T) {
 	}
 }
 
+// TODO: Rename it to `TestIntegration_BasicTypes` and remove the current one once UUID is enabled in production.
+// Test encoding/decoding non-struct Cloud Spanner types.
+func TestIntegration_UUID(t *testing.T) {
```

Comment:
nit: Can we not just have UUID test here, I see we are doing other types tests also which is duplicate?

Reply:
We can test only UUID, but the current test is extended due to couple of reasons:

1. This is a temporary test.
2. Clean-up is simple once UUID is released. Hence, the comment on function as well.